search-api/entrypoint.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+
+if [[ $ENVIRONMENT == "PRODUCTION" ]]
+then
+	echo production 
+else
+	pip install ipython
+	pip install -r requirements.txt
+	chmod -R 777 /code/
+	chown -R www-data:www-data /code/
+fi
+export FLASK_APP=search
+# flask db init
+flask db migrate
```

Comment:
migrate should not be a part of entrypoint - it creates new migrations files, which will be created by the developer. We can have upgrade.

Reply:
@vincue what about this one?

Reply:
You're right, I've made the changes. Upgrade will do the job.